libpldmresponder/platform.cpp

```diff
@@ -68,7 +72,7 @@ void Handler::generate(const pldm::utils::DBusHandler& dBusIntf,
 {
     for (const auto& directory : dir)
     {
-        std::cerr << "checking if : " << directory << "exists" << std::endl;
+        error("checking if : {DIR} exists", "DIR", directory.c_str());
```

Comment:
should this be info?

Reply:
yes, I will take this up in log improvision PR